### DISCUSSION TITLE
Add more function for different expr

### DISCUSSION THREAD

**Discussion by Superbody** [AfterShip/clickhouse-sql-parser#29] [Ideas]
The parser is great to use, but for different Exprs, more funcs are needed to get some basic info. like for SelectQuery, maybe I want GetDatabase or GetTableName funcs to ez use.

exmple:
func (s *SelectQuery) GetDatabaseString() string {
if s.From == nil {
return ""
}
x := s.From.Expr.(*TableExpr)
y := x.Expr.(*TableIdentifier)
return y.Database.String(0)
}

**Comment by git-hulk** [comment]
@Superbody Thanks for your proposal. It makes sense, would you like to submit a PR?
